packages/lesswrong/components/posts/TableOfContents/TableOfContentsList.tsx

```diff
@@ -118,7 +118,7 @@ const TableOfContentsList = ({tocSections, title, onClickSection, displayOptions
           });
         }
       }}
-      highlighted={currentSection === "above"}
+      highlighted={currentSection === "above" || currentSection === null}
```

Comment:
Another separate fix so the title starts off bolded (previously it would only be bolded after the `currentSection` had been calculated)